# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gimp/gimptool.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- gimp-2.2.8_orig/gimptool-2.0.in	2004-04-04 03:12:32.000000000 +0200
+++ gimp-2.2.8/gimptool-2.0.in	2005-07-09 23:27:50.474733976 +0200
@@ -104,7 +104,7 @@
 fi
 
 if test x${INSTALL+set} != xset ; then
-  INSTALL='@INSTALL@'
+  INSTALL='install'
   if test "$INSTALL" = "./install-sh -c"; then
     mydirname=`echo $0 | sed -e 's#\(.*\)/[^/].*$#\1#'`
     INSTALL="$mydirname/gimpinstall"
